src/app/utils/graph-toolkit-lookup.ts

```diff
@@ -1,7 +1,33 @@
+import { SeverityLevel } from '@microsoft/applicationinsights-web';
 import templates from '../../graph-toolkit-examples';
+import { telemetry } from '../../telemetry';
+import { LINK_ERROR } from '../../telemetry/error-types';
 import { IQuery } from '../../types/query-runner';
+import { sanitizeQueryUrl } from './query-url-sanitization';
 import { parseSampleUrl } from './sample-url-generation';
 
+
+export async function validateToolkitUrl(url: string, sampleQuery: IQuery): Promise<void> {
```

Comment:
Is this the only scenario for which we'd want to validate URLs (now and into the future)? I suggest pulling this out of this file and make this generally available for all components to validate URLs.

Reply:
I've moved the method to a separate file.